cockroachdb/manifests-cert-manager/certificates-patch.yaml

```diff
@@ -87,3 +72,27 @@ spec:
                     - key: tls.key
                       path: client.root.key
             defaultMode: 256
+---
```

Comment:
I don't want to be a pain but how about getting rid of `certificates-patch.yaml` and do these path overrides directly on the resource(Job, StatefulSet)

Reply:
That's really good idea, I don't know why I didn't thought about that :) 
[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/48/commits/4e388857ec8a5c22fc27e044674f21820c3d9123)